dev/.docker/goaws.yaml

```diff
@@ -0,0 +1,10 @@
+Dev:                                # Another environment
+  Host: goaws
+  Port: 4100
+  # SqsPort: 9324
+  # SnsPort: 9292
```

Comment:
Why include these if we are not using them?

Reply:
Fixed in last commit